01_P/P_2_1_1_01/sketch.js

```diff
@@ -18,34 +18,33 @@
 
 /**
  * changing strokeweight and strokecaps on diagonals in a grid
- *   
+ *
  * MOUSE
  * position x          : left diagonal strokeweight
  * position y          : right diagonal strokeweight
  * left click          : new random layout
- * 
+ *
  * KEYS
  * 1                   : round strokecap
  * 2                   : square strokecap
  * 3                   : project strokecap
  * s                   : save png
  */
+'use strict';
 
 var tileCount = 20;
 var actRandomSeed = 0;
 
 var actStrokeCap;
 
-
 function setup() {
   createCanvas(600, 600);
+  noFill();
   actStrokeCap = ROUND;
 }
 
 function draw() {
   background(255);
-  smooth();
```

Comment:
Removed smooth() since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).